content/docs/pipelines/sdk/output-viewer.md

```diff
@@ -280,9 +297,16 @@ assumes that the schema includes three columns with the following names:
 * `tpr` (true positive rate)
 * `thresholds`
 
+**Optional metadata fields:**
+
+- `storage`
+
 When viewing the ROC curve, you can hover your cursor over the ROC curve to see 
 the threshold value used for the cursor's closest `fpr` and `tpr` values.
 
+You can optionally specify `'storage': 'inline'` to embed raw content of roc_file
```

Comment:
Similar comment to `cm_file`, replace `roc_file` with:

the ROC curve file

and 

[ROC_FILE]

Reply:
`roc_file` below is a variable in the snippet, so I didn't rename it